modules/mobile/docs/schemas/RefillPayload.yml

```diff
@@ -0,0 +1,7 @@
+type: object
```

Comment:
I'm not sure why we habitually extract these into schemas rather than just do them inline within the openapi.yaml. I'm not requesting a change here, but my general expectation would be that these schemas should only be for things we expect to use multiple endpoints. NBD though. This is perfectly correct.

Reply:
My Rubymine will get super slow and eventually crash if I keep this openapi.yaml open too long. so it may be to keep the file from exploding more then it has too. but thats probably more a rubymine issue that should be solved.